src/QMCTools/CMakeLists.txt

```diff
@@ -47,6 +50,15 @@ ADD_EXECUTABLE(qmc-check-affinity check-affinity.cpp)
 INSTALL(TARGETS qmc-check-affinity
         RUNTIME DESTINATION bin)
 
+ADD_EXECUTABLE(QBoxConverter QBoxConverter.cpp XmlRep.cpp WriteEshdf.cpp)
```

Comment:
To follow the pattern, the name of the converter should have qmc in it.   Maybe qmc-qbox-converter ?